### DISCUSSION TITLE
CSA Night at the Museum Checkpoint

### DISCUSSION THREAD

**Discussion by giscus** [YLu-1258/YLU_blog#8] [Comments]
# CSA Night at the Museum Checkpoint

Two Trimester Focus Planning

https://ylu-1258.github.io/YLU_blog/posts/MuseumReflection/

**Comment by YLu-1258** [comment]
test

**Comment by jm1021** [comment]
I would like to see your group show me pull requests in the future.   It is very important to have a person to be in charge of integrations and reviews.   The may be required to revert as well.
